src/library/scala/util/hashing/MurmurHash3.scala

```diff
@@ -70,7 +74,7 @@ private[hashing] class MurmurHash3 {
       x.productPrefix.hashCode
```

Comment:
Presumably this line should also be adapted:
```suggestion
      (if (caseClassName != null) caseClassName else x.productPrefix).hashCode
```